topics/csg/index.md

```diff
@@ -0,0 +1,11 @@
+---
+aliases:  constructive-solid-geometry, constructive-solid-geometries
+display_name: Constructive Solid Geometry
+related: 3d, raytracer, rendering, bsp
+short_description: CSG is short for the Constructive Solid Geometry, a modelling technique that combine 3D solids.
```

Comment:
I don't think we need the `the` article here. Also, I believe that `modeling` is spelt with 1 `l`.
```suggestion
short_description: CSG is short for Constructive Solid Geometry, a modeling technique that combine 3D solids.
```